src/mlpack/tests/random_forest_test.cpp

```diff
@@ -607,3 +607,140 @@ TEST_CASE("ExtraTreesAccuracyTest", "[RandomForestTest]")
 
   REQUIRE(accuracy >= 0.85);
 }
+
+/**
+ * Test ComputeAverageUniqueness.
+ * 
+ * Average uniqueness is defined as average over the lifetime of an event
+ * of how many other events are active at every point in time.
+ */
+TEST_CASE("ComputeAverageUniquenessTest", "[RandomForestTest]")
+{
+  arma::umat indM(2 /* rows */, 3 /* cols */, arma::fill::zeros);
+
+  // indM = [1 1 1 0 0 0
+  //         0 0 1 1 0 0
+  //         0 0 0 0 1 1]
+  // The last event is fully isolated.
+  // The first two events overlap in the third column.
+  indM(0, 0) = 0;
+  indM(1, 0) = 2;
+  indM(0, 1) = 2;
+  indM(1, 1) = 3;
+  indM(0, 2) = 4;
+  indM(1, 2) = 5;
+
+  arma::vec concurrency(6);
+  concurrency[0] = 1;
+  concurrency[1] = 1;
+  concurrency[2] = 2;
+  concurrency[3] = 1;
+  concurrency[4] = 1;
+  concurrency[5] = 1;
+
+  const arma::vec avg(SequentialBootstrap<>::ComputeAverageUniqueness(indM,
+    arma::linspace<arma::uvec>(0, 2, 3), concurrency));
+
+  REQUIRE(avg.n_rows == 3);
+  REQUIRE(avg(0) == 5.0 / 6.0);
+  REQUIRE(avg(1) == 0.75);
+  REQUIRE(avg(2) == 1.0);
```

Comment:
```suggestion
  REQUIRE(avg(0) == Approx(5.0 / 6.0));
  REQUIRE(avg(1) == Approx(0.75));
  REQUIRE(avg(2) == Approx(1.0));
```
This is to handle floating point inaccuracies.